Source/AssistantV1/Models/DialogNode.swift

```diff
@@ -23,7 +23,7 @@ public struct DialogNode: Codable, Equatable {
     /**
      How the dialog node is processed.
      */
-    public enum NodeType: String {
+    public enum TypeEnum: String {
```

Comment:
Looks like we now generate multiple `TypeEnum` enums.  We should make sure that these are usable, by e.g. qualifying them with the struct name.

Reply:
I believe that all of the `TypeEnum` definitions are properties of a containing struct. I am not sure that we will have collision here, and all test cases build correctly.